manifests/w/WindRiver/studio-cli/23.9.1/WindRiver.studio-cli.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=NVS1.CRLF.5-1-22621-1778.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WindRiver.studio-cli
```

```suggestion
PackageIdentifier: WindRiver.StudioCLI
```
There is already an existing identifier for this package

---

Hello, thank you very much for your suggestion. If modifying this identifier causes the validation pipeline to fail and prompts for changes to 'studio-cli' in the path 'winget-pkgs\manifests\w\WindRiver\studio-cli', and previous versions in the 'studio-cli' directory also used 'PackageIdentifier: WindRiver.studio-cli', is it possible to leave it unchanged?